### PR TITLE
Global App Navigation button icon added

### PR DESCRIPTION
**Pull request by joachim-junior**
Things implemented.

The app-nav button is visible in the top right of the screen in the global header
Clicking the app-nav button opens a drop-down with options(These options are being implemented by another Ticket). So right now just the dropdown opens. 
Clicking "X" should close the menu.

### CODE REVIEW

**Comment by joachim-junior** [comment]
Things implemented.
- The app-nav button is visible in the top right of the screen in the global header
- Clicking the app-nav button opens a drop-down with options.
- Clicking "X" should close the menu.

File: src/urls.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const SourceFiles = {
+  clinical: "home",
+  patients: "patient-registration",
```
Comment (FlorianRappl) [inline src/urls.tsx]: Owned by `@openmrs/esm-patient-registration-app`

File: src/urls.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const SourceFiles = {
+  clinical: "home",
```
Comment (FlorianRappl) [inline src/urls.tsx]: Owned by `@openmrs/esm-home-app`

File: src/components/navbar/navbar.component.tsx [hunk 3]
```diff
@@ -94,6 +118,60 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
```
Comment (FlorianRappl) [inline src/components/navbar/navbar.component.tsx:115]: Most of these already exist.

Also they should all use the existing extension slot.

File: src/components/navbar/navbar.component.tsx [hunk 4]
```diff
@@ -4,17 +4,25 @@ import UserMenuPanel from "../navbar-header-panels/user-menu-panel.component";
...
+  HeaderPanel,
+  Switcher,
+  SwitcherItem,
+  SwitcherDivider
 } from "carbon-components-react/es/components/UIShell";
 import { LoggedInUser, UserSession } from "../../types";
 import styles from "./navbar.scss";
 import { getCurrentSession } from "../../root.resource";
+import SourceFiles from "../../urls";
+import { HeaderSideNavItems } from "carbon-components-react";
```
Comment (FlorianRappl) [inline src/components/navbar/navbar.component.tsx]: Use selective imports as shown above for the UIShell. Don't use the full package.

**Review by FlorianRappl — CHANGES_REQUESTED** [review]

**Comment by joachim-junior** [comment]
Hi @FlorianRappl, I noticed I did some work on this ticket which is out of scope. The options available is implemented by another ticket. Please you can see that [Here](https://issues.openmrs.org/browse/MF-465).

**Comment by joachim-junior** [comment]
So I will Remove that aspect on this ticket and maintain the scope.

**Comment by joachim-junior** [comment]
I could start working on the other issue

**Pull request closed without merging**